src/router/storeSync/layersParamParser.js

```diff
@@ -109,3 +113,22 @@ export function transformLayerIntoUrlString(layer, defaultLayerConfig) {
     }
     return layerUrlString
 }
+
+/**
+ * @param {SelectableFeature[]} selectedFeatures An array of selectable Features
+ * @returns {Object} A simple object with the layer id as a key for a feature Ids list as value
+ */
+export function orderFeaturesByLayers(selectedFeatures) {
+    const layersFeatures = {}
+
+    selectedFeatures.forEach((feature) => {
+        if (!layersFeatures[feature.layer.id]) {
+            layersFeatures[feature.layer.id] = []
+        }
+        // In some features, the 'id' get attribute is built by concatening {layer.id}-{feature.id}
+        // We remove this concatenation here.
```

Comment:
Here the comment seems out of date ? Or some code is missing ?

Reply:
It is indeed an outdated comment